docs/configure/agent.md

```diff
@@ -0,0 +1,240 @@
+
+1. Create a configuration file with the desired configuration for your machine. You can:
+
+   - [create a new machine in the Viam app](/cloud/machines/#add-a-new-machine) and configure it as desired or use an existing deployed machine, then switch to **Raw JSON** mode and copy the configuration shown into a new file on your machine.
+   - base your configuration on the [example configuration file](/internals/local-configuration-file/#example-json-configuration-file), and adjust as needed.
+
+1. Place the configuration into a file called <file>viam.json</file> in the <file>/etc/</file> folder on the machine you wish to install `viam-agent` to: <file>/etc/viam.json</file>.
+
+1. Run the following command to install `viam-agent` on your machine:
+
+   ```sh {class="command-line" data-prompt="$"}
+   sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/install.sh)"
+   ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+`viam-agent` will install itself as a systemd service named `viam-agent`.
+For information on managing the service, see [Manage `viam-agent`](/installation/manage-viam-agent/).
+
+## Configuration
+
+{{< tabs >}}
+{{% tab name="Default" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "agent": {
+    "viam-agent": {
+      "release_channel": "stable",
+      "pin_version": "",
+      "pin_url": "",
+      "disable_subsystem": false
+    },
+    "viam-server": {
+      "release_channel": "stable",
+      "pin_version": "",
+      "pin_url": "",
+      "disable_subsystem": false,
+      "attributes": {
+        "fast_start": false
+      }
+    },
+    "agent-provisioning": {
+      "release_channel": "stable",
+      "pin_version": "",
+      "pin_url": "",
+      "disable_subsystem": false
+    },
+    "agent-syscfg": {
+      "release_channel": "stable",
+      "pin_version": "",
+      "pin_url": "",
+      "disable_subsystem": false
+    }
+  }
+}
+```
+
+{{% /tab %}}
+{{% tab name="Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "agent": {
+    "viam-agent": {
+      "release_channel": "stable",
+      "pin_version": "1.2.3",
+      "pin_url": "http://example/test.binary",
+      "disable_subsystem": false
+    },
+    "viam-server": {
+      "attributes": {
+        "fast_start": true
+      }
+    },
+    "agent-provisioning": {
+      "release_channel": "stable"
+    },
+    "agent-syscfg": {
+      "release_channel": "stable"
+    }
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Each section primarily controls updates for that subsystem:
+
+### `viam-agent`
+
+<!-- prettier-ignore -->
+| Option | Type | Required? | Description |
+| ------ | ---- | --------- | ----------- |
+| `release_channel` | string | Optional | `viam-agent` is semantically versioned and is tested before release. Releases happen infrequently. When set to `"stable"`, `viam-agent` will automatically upgrade when a new version is released. Options: `"stable"` (default), `"??"`. |
```

Comment:
```suggestion
| `release_channel` | string | Optional | `viam-agent` is semantically versioned and is tested before release. Releases happen infrequently. When set to `"stable"`, `viam-agent` will automatically upgrade when a new version is released. Options: `"stable"` (default), `"??"`. |
```
what is ??

Reply:
An indication that I need the SME to tell me what to put here.

Reply:
```suggestion
| `release_channel` | string | Optional | `viam-agent` is semantically versioned and is tested before release. Releases happen infrequently. When set to `"stable"`, `viam-agent` will automatically upgrade when a new version is released. Options: `"stable"` (default). |
```